src/abft-state-transitions-new-step.md

```diff
@@ -0,0 +1,38 @@
+$$
+\newcommand \FilterTimeout {\mathrm{FilterTimeout}}
+\newcommand \DeadlineTimeout {\mathrm{DeadlineTimeout}}
+\newcommand \Cert {\mathit{cert}}
+\newcommand \Next {\mathit{next}}
+$$
+
+# New Step
+
+A player may also update its step after receiving a timeout event.
+
+On observing a timeout event of \\( \FilterTimeout(p) \\) for a period \\( p \\), the
+player sets \\( s := \Cert \\).
+
+On observing a timeout event of \\( \DeadlineTimeout(p) \\) for a period \\( p \\), the
+player sets \\( s := \Next_0 \\).
+
+On observing a timeout event of \\( \DeadlineTimeout(p) + 2^{s_t}\lambda + u \\) where
+\\( u \in [0, 2^{s_t}\lambda) \\) sampled uniformly at random, the player sets
+\\( s := s_t = \Next_{s_t-3} \\).
```

Comment:
@cce @jannotti this is an enhancement wrt the original specs.

Reply:
We restored the original notation in https://github.com/algorandfoundation/specs/pull/125/commits/43d41524f422f01933660353af17c4f47f2109c9 and will add just a comment instead.